hwbench/environment/vendors/pdus/generic.py

```diff
@@ -59,14 +65,18 @@ def get_power(self):
             option, path = self.outlet, "Outlets"
         if self.multioutlet:
             for opt in option.split(","):
-                power.append(self.get_redfish_url(f"{self.redfish_root}{path}/{opt}/"))
+                power.append(self.get_power_outlet(f"{self.redfish_root}{path}/{opt}"))
         else:
-            power.append(self.get_redfish_url(f"{self.redfish_root}{path}/{option}/"))
+            power.append(self.get_power_outlet(f"{self.redfish_root}{path}/{option}"))
         return power
 
     def get_power_total(self):
         total = 0.0
         for outlet in self.get_power():
+            if "PowerWatts" not in outlet or "Reading" not in outlet["PowerWatts"]:
+                h.fatal(
+                    f"Outlet for {self.get_url()} has no power: {outlet}\noutlet={self.outlet}, outletgroup={self.outletgroup}"
```

Comment:
"has no power" could be misleading as we don't speak about the PDU power but the presence of  "power metrics".
I suggest "Outlet .... do not expose power metrics"